tests/lights/test_light.py

```diff
@@ -298,9 +347,25 @@ async def test_enum_light_properties_no_key(deconz_light):
     """Verify enum properties return expected values or None."""
     light = await deconz_light({"config": {}, "state": {}, "type": "Color light"})
 
+    assert light.supported_effects is None
     assert light.alert is None
     assert light.color_capabilities is None
     assert light.color_mode is None
     assert light.effect is None
     with pytest.raises(KeyError):
         assert light.fan_speed
+
+
+ENUM_CAPABILITIES_DATA = [
+    ({"capabilities": {}}),
+    ({"capabilities": {"color": {}}}),
+]
+
+
+@pytest.mark.parametrize(("data"), ENUM_CAPABILITIES_DATA)
+async def test_enum_light_properties_no_capabilities(deconz_light, data):
+    """Verify enum properties return expected values or None."""
+    light = await deconz_light({"config": {}, "state": {}, "type": "Color light"})
+
+    light.update(data)
```

Comment:
No need to test the update method?
```suggestion
    light = await deconz_light({"config": {}, "state": {}, "type": "Color light"} | data)
```

Reply:
I don't think so. It's the same method `test_enum_light_properties` and `test_light_light` are calling to update the state of the light test fixture. I'm not sure how one would go about writing a test for that.

Reply:
That's neat! I didn't think to do that.